test/tests/functional/pbs_job_array.py

```diff
@@ -133,6 +133,7 @@ def kill_and_restart_svr(self):
         rv = self.is_server_licensed(self.server)
         _msg = 'No license found on server %s' % (self.server.shortname)
         self.assertTrue(rv, _msg)
+        self.mom.isUp()
```

Comment:
isUp() returns True/False if the mom is up or not.  If the mom fails to come up, just waits some time and returns false.  Since you are not checking for that, it might not work.  I'd put this in a self.assertTrue()

Reply:
Hi @bhroam While working on this, I realized that this case does not involve mom restart; but involves only server restart. Hence, node state check would suffice here and check for mom pid is not necessary. I have updated the same in the code. Please have a look and let me know your view on this.